### ISSUE TITLE
Provide vdrift-data on github.com or other git hoster

### ISSUE THREAD

**Issue by krichter722** [VDrift/vdrift#146]
It'd be nice to have `vdrift-data` from https://sourceforge.net/p/vdrift/code/HEAD/tree/ at github.com or as `git` some other way.

**Comment by logzero** [comment]
Yeah, this has been discussed a number of times. If it was just a few hundred MBs, it would have happened already. We are talking about ~3.5GB binary data though.

See also https://help.github.com/articles/what-is-my-disk-quota/

So the motivation from my side is not very high. ;)

**Comment by Bengt** [comment]
@krichter722 Hosting vdrift-data on GitHub was discussed in <https://github.com/Bengt/VDrift-Data/issues/2>, for example. I basically dumped a SVN checkout into a Git repository to evaluate that idea. Although this worked in principle, there were some issues that could not be resolved completely. However, Git now has way better [support for large binary files](https://git-lfs.github.com/), making a Git-based workflow much more viable.

**Comment by Bengt** [comment]
I imported VDrift using GitHub's importer: <https://github.com/Bengt/VDrift-2016-07-09> VDrift is actually that small (1.81 GB) in Git that it can reside in a single repository. Given today's broadband internet connections and cheap SSDs, splitting the repository up into multiple submodules does not seem worth the hassle and neither does using (and paying for) LFS. I suspect that enabling collaboration via pull requests is more important than being able to work with individual assets, nowadays.

**Comment by Apteryks** [comment]
I've tried to checkout the latest vdrift-data using subversion from sourceforge, but it failed like:
```
svn: E175002: Unexpected HTTP status 503 'Service Unavailable' on '/p/vdrift/code/!svn/rvr/910/vdrift-data/tracks/bahrain/objects/roadd.dds-misc1.png'
svn: E200042: Additional errors:
svn: E175002: Unexpected HTTP status 503 'Service Unavailable' on '/p/vdrift/code/!svn/rvr/178/vdrift-data/tracks/bahrain/objects/obj_trucka_bms.dds.png'
```

**Comment by Apteryks** [comment]
Entering the svn checkout directoring and issuing `svn clean && svn up` seems to have resumed the download.
